Pod/Classes/Data.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Data.swift
+//  Pods
+//
+//  Created by Adam Galloway on 10/7/15.
+//
+//
+
+import Foundation
+
+public final class Data: Model,ResponseObject,ResponseCollection {
+
+    public let group: String
```

Comment:
I used let, so that the developers can't change the value after they fetch it, since there is no way to save the change, same with uuid (they're immutable)